docs/source/documentation/patterns/sample-code/_accordions.html

```diff
@@ -1,35 +1,38 @@
-<aside class="js-accordion accordion">
-  <div class="accordion-section is-active">
+<aside class="js-accordion accordion {{modifier}}">
+  <div class="accordion-section">
     <h4 class="accordion-title">
       <span class="accordion-icon">
         <svg xmlns="http://www.w3.org/2000/svg" width="14" height="14" viewBox="0 0 32 32" class="svg-icon"><path d="M28 9v5L16 26 4 14V9l12 12L28 9z"/></svg>
       </span>
       Title of Section
+      <div class="accordion-content">
+        <p>Stuff!</p>
+      </div>
     </h4>
-    <div class="accordion-content">
-      <p>Stuff!</p>
-    </div>
   </div>
-  <div class="accordion-section">
+  <div class="accordion-section is-active">
     <h4 class="accordion-title">
       <span class="accordion-icon">
         <svg xmlns="http://www.w3.org/2000/svg" width="14" height="14" viewBox="0 0 32 32" class="svg-icon"><path d="M28 9v5L16 26 4 14V9l12 12L28 9z"/></svg>
       </span>
       Title of Second Section
+      <div class="accordion-content">
+        <p>Some long stuff!</p>
+        <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Vestibulum in erat eros. Aliquam non pulvinar diam. Nulla pharetra eros vel turpis varius, at hendrerit nulla ultrices. Vestibulum pretium ac mauris vitae sodales. Mauris tincidunt felis ac orci accumsan, at lobortis eros scelerisque. Etiam egestas libero sed lacinia cursus. Vestibulum auctor efficitur elit. Donec gravida urna quis urna blandit porttitor. Quisque vel tristique ipsum. Phasellus consequat aliquam odio, sed efficitur est gravida viverra. Duis quis nibh hendrerit, auctor dolor sit amet, placerat mauris. Suspendisse viverra urna et est consectetur feugiat vel vel est. Nam lorem leo, elementum sed quam id, placerat convallis massa.</p>
+        <div class="btn btn-primary">Button</div>
+      </div>
     </h4>
-    <div class="accordion-content">
-      <p>Stuff!</p>
-    </div>
   </div>
   <div class="accordion-section">
     <h4 class="accordion-title">
       <span class="accordion-icon">
         <svg xmlns="http://www.w3.org/2000/svg" width="14" height="14" viewBox="0 0 32 32" class="svg-icon"><path d="M28 9v5L16 26 4 14V9l12 12L28 9z"/></svg>
       </span>
       Title of Third Section
+      <div class="accordion-content">
+        <p>Stuff!</p>
+      </div>
     </h4>
-    <div class="accordion-content">
-      <p>Stuff!</p>
-    </div>
   </div>
-</aside>
\ No newline at end of file
+</aside>
+
```

Comment:
Is this change of structure intentional? This makes the entirety of the content area clickable to toggle closing the section. Currently you can only click the title to toggle opening and closing in CW.

Reply:
updated